.devops/code-review-pipelines.yml

```
@@ -1,59 +1,105 @@
+              sonar.coverage.jacoco.xmlReportPaths=./target/site/jacoco/jacoco.xml
+              sonar.junit.reportPaths=target/surefire-reports/
+
+        - bash: |
+              docker run -p 10000:10000 -p 10001:10001 -p 10002:10002 mcr.microsoft.com/azure-storage/azurite &
+          displayName: Run azurite to test Service - Storage integration
+
+        - task: Maven@3
+          inputs:
+            mavenPomFile: 'pom.xml'
+            mavenOptions: '-Xmx3072m $(MAVEN_OPTS)'
+            javaHomeOption: 'JDKVersion'
+            jdkVersionOption: '17'
+            jdkArchitectureOption: 'x64'
+            publishJUnitResults: false
+            testResultsFiles: '**/surefire-reports/TEST-*.xml'
+            goals: 'clean verify'
+            sonarQubeRunAnalysis: true
+
+        - task: SonarCloudPublish@1
+          displayName: 'Publish SonarCloud results on build summary'
+          inputs:
+            pollingTimeoutSec: '300'
+
+        - task: PublishCodeCoverageResults@1
+          inputs:
+            codeCoverageTool: 'JaCoCo'
+            summaryFileLocation: 'target/site/jacoco/jacoco.xml'
+            reportDirectory: 'target/site/jacoco'
+          displayName: 'Publish Code Coverage on Azure Devops'
+
+  - stage: IntegrationTest
+    dependsOn: []
+    jobs: 
+      - job: make_integration_test  
+        steps:
+        - checkout: pagopaEcommerceLocal
+        - script: | 
+            echo "##[debug] Change placeholder for ecommerce-transaction-service ..."
+            sed -i.bak  "s/\${ECOMMERCE_TRANSACTIONS_COMMIT_SHA}/$(BRANCH_NAME)/" global-transaction-service.env 
+            cp global-transaction-service.env .env 
+            rm global-transaction-service.env.bak 
+          displayName: 'Change transaction branch name ecommerce-local .env'
+        - script: docker-compose up --scale mongo-express=0 --scale redis-insight=0 --scale pagopa-notifications-service=0 -d
+          displayName: 'Run app with docker-compose'
+        - script: |
+              timeout 180 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' localhost:8080/actuator/health/liveness)" != "200" ]]; do sleep 5; done' || exit 1
+          displayName: 'Health check'
+        - script: yarn global add newman
+          displayName: 'newman installation'
+        - script: newman run api-tests/ecommerce-for-checkout/transaction-service/transaction-service_ecommerce_local.postman_collection.json --environment=api-tests/ecommerce-for-checkout/transaction-service/transaction-service_ecommerce_local.postman_environment.json --reporters cli,junit --reporter-junit-export Results/checkout-api-TEST.xml
+          displayName: 'Run api test'
+          continueOnError: true 
+        - task: PublishTestResults@2
+          inputs:
+            testResultsFormat: 'JUnit'
+            testResultsFiles: '**/*-TEST.xml' 
+            searchFolder: '$(System.DefaultWorkingDirectory)'
+      
+
+
+
+
+
```

```suggestion
            searchFolder: '$(System.DefaultWorkingDirectory)'

```

Formatting (also newlines)